backend/dal/Migrations/v01.07.00/Down/PostDown/01-ViewProperties.sql

```diff
@@ -0,0 +1,219 @@
+﻿PRINT 'Updating View_Properties'
+GO
```

Comment:
@goemen I merged my changes, so You'll need to update your up/down based on my ViewProperties.sql